### ISSUE TITLE
HTTP 500 response from GET http://127.0.0.1:44585/music/audioaddict/service?serv=di

### ISSUE THREAD

**Issue by NKnusperer** [phrawzty/AudioAddict.bundle#34]
DI.fm stopped working today ("This channel is not responding.").
When requesting `/music/audioaddict/service?serv=di`  this is the response:
```
<Response code="2000" status="ValueError: No JSON object could be decoded">
<Traceback>
Traceback (most recent call last): File "/usr/lib/plexmediaserver/Resources/Plug-ins-25d94bad9/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/runtime.py", line 843, in handle_request result = f(**d) File "/config/Library/Application Support/Plex Media Server/Plug-ins/AudioAddict.bundle/Contents/Code/__init__.py", line 72, in GetChannels for channel in AA.get_batchinfo(refresh=True): File "/config/Library/Application Support/Plex Media Server/Plug-ins/AudioAddict.bundle/Contents/Code/audioaddict.py", line 300, in get_batchinfo batch = json.loads(data) File "/usr/lib/plexmediaserver/Resources/Python/lib/python2.7/json/__init__.py", line 339, in loads return _default_decoder.decode(s) File "/usr/lib/plexmediaserver/Resources/Python/lib/python2.7/json/decoder.py", line 364, in decode obj, end = self.raw_decode(s, idx=_w(s, 0).end()) File "/usr/lib/plexmediaserver/Resources/Python/lib/python2.7/json/decoder.py", line 382, in raw_decode raise ValueError("No JSON object could be decoded") ValueError: No JSON object could be decoded
</Traceback>
</Response>
```

**Comment by phrawzty** [comment]
Yeah, it looks like all of the AA API endpoints are behaving this way. I'll try to take a look this weekend. Thanks for letting me know!

**Comment by phrawzty** [comment]
As it turns out, the AA API has implemented gzip compression for one (and so far _only_ one) of the responses. I've pushed a fix for this in #36, and will prepare release `0.6.2` momentarily. This should fix the errant behaviour you're seeing (it worked for me!) - if not, please let me know.
